packages/design/src/DataTable/Cells.tsx

```diff
@@ -97,20 +99,22 @@ export const ClickableLabelCell = ({
   labels: LabelDescription[];
   onClick: (label: LabelDescription) => void;
 }) => {
-  const $labels = labels.map(label => (
-    <Label
-      onClick={() => onClick(label)}
-      key={`${label.name}:${label.value}`}
-      mr="1"
-      mb="1"
-      kind="secondary"
-      css={`
-        cursor: pointer;
-      `}
-    >
-      {`${label.name}: ${label.value}`}
-    </Label>
-  ));
+  const $labels = labels
+    .filter(label => !label.name.startsWith('teleport.internal'))
```

Comment:
Since this component is part of the design package we may don't want to put teleport-specific business logic inside. The design only provides UI components and the consumer package should shape the data to the correct form. 
On the other hand, this place will ensure that all cases are handled. What do you think?

Reply:
Another option would be to filter it inside data fetching services. Which of course can be done. But would require a bit of duplication to have it applied for each service. Which is fine, but I don't think it is necessary.

Reply:
Alternatively, we could create `TeleportLabelCell` wrapper for `ClickableLabelCell` and put it in the `shared` package:
```tsx
export const TeleportLabelCell = ({
  labels,
  ...props
}: ClickableLabelCellProps) => (
   <ClickableLabelCell
      {...props}
      labels={labels.filter(({ name }) => !name.startsWith('teleport.internal'))}
  />
);
```
And use it instead of ClickableLabelCell where needed.

Reply:
That's actually a great idea. I would put it into teleport/components as it is actually used and needed only inside teleport package